CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 * Remove the autonaming of `vault.identify.GroupAlias`. The `name` must match that of the Github Team Name.
 * Namespace names in .NET SDK are adjusted to PascalCase
 ([#17](https://github.com/pulumi/pulumi-vault/pull/17)).
+* **BREAKING CHANGE:** `vault.Token` module has been renamed to `vault.TokenAuth` this is to avoid a conflict between Token class and Token module
```

Comment:
Should we list this at the top of the CHANGELOG in its own section, and make that a standard for CHANGELOGs going forward?

Reply:
Agreed we can do that for sure - I will shuffle this before the release